nexus/src/internal_api/http_entrypoints.rs

```diff
@@ -87,6 +88,35 @@ async fn cpapi_sled_agents_post(
     apictx.internal_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Path parameters for Rack requests.
+#[derive(Deserialize, JsonSchema)]
+struct RackPathParam {
+    rack_id: Uuid,
+}
+
+/// Report that the Rack Setup Service initialization is complete
+///
+/// See RFD 278 for more details.
+#[endpoint {
+     method = PUT,
+     path = "/racks/{rack_id}/initialization_complete",
+ }]
+async fn rack_initialization_complete(
```

Comment:
This is the API I expect RSS to invoke.

I'll submit the call to it in a follow-up PR, but wanted this one to be focused on the Nexus side of things.

Reply:
Let me make sure I understand the sequencing. RSS configures a bunch of services to run on various sleds, among which is `nexus`. This represents the handshake between RSS and `nexus`, at which point `nexus` takes over control of the rack, is that right?

Reply:
There's a lot of detail on this handshake in RFD 278, if you want more context. But yeah, that description is accurate.

In the future, I expect Nexus to refuse to serve the external interface until this method succeeds - so if it fails, the internal API would remain accessible, but the external API would stay offline. I think this scenario would require operator intervention, but would be very obvious as it would happen during rack setup.